### PR TITLE
Clarify our methods of authenticating to Grafana

### PR DESCRIPTION
**Pull request by sgibson91**
- fixes https://github.com/2i2c-org/infrastructure/issues/3505

Note, the `grafana-access:invite-link` relative link will work when #3512 is merged

----
:books: Documentation preview :books:: https://2i2c-pilot-hubs--3513.org.readthedocs.build/en/3513/

### CODE REVIEW

File: docs/howto/grafana-github-auth.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 We can enable GitHub authentication against a Grafana instance in order to allow access to the dashboards for hub administrators as well as 2i2c engineers.
 
+```{note}
+We only offer this method of authentication to communities if they want to give
```
Comment (GeorgianaElena) [inline docs/howto/grafana-github-auth.md:7]: While reading this I was thinking it might be worth nothing that we use this by default to grant access to all 2i2c staff into Grafanas. Maybe somthing like:

```suggestion
This is the default authentication method for 2i2c staff wanting to visualize the dashboards, but we can also offer this method of authentication to communities only if they want to give
```

File: docs/topic/monitoring-alerting/grafana.md [hunk 2]
```diff
@@ -3,27 +3,36 @@
...
 
 Each cluster's Grafana deployment can be accessed at `grafana.<cluster-name>.2i2c.cloud`.
-For example, the Grafana for the community hubs running on our GCP project is accessible at `grafana.pilot.2i2c.cloud`. Checkout the list of all 2i2c running clusters and their Grafana [here](https://infrastructure.2i2c.org/en/latest/reference/hubs.html).
+For example, the Grafana for the community hubs running on our GCP project is accessible at `grafana.pilot.2i2c.cloud`.
+Checkout the list of all 2i2c running clusters and their Grafana [here](https://infrastructure.2i2c.org/en/latest/reference/hubs.html).
 
 To access the Grafana dashboards you have two options:
 
-- Get `Viewer` access into the Grafana.
+- Authenticate with GitHub to get `Viewer` access into the Grafana, _if enabled_.
```
Comment (GeorgianaElena) [inline docs/topic/monitoring-alerting/grafana.md:22]: I was thinking that turning this into sub-headers ### might make them more visible and linkable when needed? Like:

```suggestion
### Get `Viewer` access

Authenticate with GitHub to get `Viewer` access into the Grafana, _if enabled_.
```

What do you think

File: docs/topic/monitoring-alerting/grafana.md [hunk 3]
```diff
@@ -3,27 +3,36 @@
...
 
   This is the recommended way of accessing grafana if modifying/creating dashboards is not needed.
   To get access, ask a 2i2c engineer to enable **GitHub authentication** following [](grafana-dashboards:github-auth) for that particular Grafana (if it's not already) and allow you access.
 
-- Use a **username** and **password** to get `Admin` access into the Grafana.
+- Use the **`admin` username and password** to get `Admin` access into the Grafana.
```
Comment (GeorgianaElena) [inline docs/topic/monitoring-alerting/grafana.md:27]: Same here:
```suggestion
### Get `Admin` access
Use the **`admin` username and password** to get `Admin` access into the Grafana.
```

**Review by GeorgianaElena — APPROVED** [review]
🚀 Thank you very much @sgibson91!

File: docs/howto/grafana-github-auth.md (hunk 1 above)
Comment (sgibson91) [reply docs/howto/grafana-github-auth.md:7]: Yeah. I think to me it is not clear that enabling GitHub auth on a grafana instance and granting 2i2c org access to it actually is the default/something we do every time we deploy _a new cluster_.

I think what we _actually_ do is add the new cluster to the central grafana, and we have GitHub auth for the 2i2c org there.

There's a usage question here of pulling up the specific cluster grafana URL vs finding it in the central grafana. Which path is most used? Do we expect to have GitHub auth for 2i2c enabled on all cluster grafanas, because I don't think it is?

So I think I would reword this further to: "This is the default authentication method for 2i2c staff wanting to visualise the dashboards _on the 2i2c central grafana_. However, we can also offer this method of communication..."

What do you think?

Comment (GeorgianaElena) [reply docs/howto/grafana-github-auth.md:7]: @sgibson91, good point! +1 on going with your suggested rewording 👍🏼

Comment (GeorgianaElena) [reply docs/howto/grafana-github-auth.md:7]: Thank you!
